dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/AbstractSummaryHolderJdbcConnector.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2022 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import javax.annotation.Nonnull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author shevek
+ */
+public abstract class AbstractSummaryHolderJdbcConnector extends AbstractJdbcConnector {
```

Comment:
Does this class need to go into the inheritance hierarchy? Given that it has state, might it be better as a Support-style object which is created during the task/connector runtime?